pkg/rpc/request/param.go

```diff
@@ -164,6 +164,24 @@ func (p *Param) GetUint160FromAddressOrHex() (util.Uint160, error) {
 	return p.GetUint160FromAddress()
 }
 
+// GetArrayUint160FromHex returns array of Uint160 values of the parameter that
+// was supply as array of raw hex.
+func (p *Param) GetArrayUint160FromHex() ([]util.Uint160, error) {
```

Comment:
We can check if `p == nil` here and `nil, nil` in this case. This way parameter handling is less verbose (just `arr, err := p.GetArrayUint160FromHex` instead of 1 or 2 if's.